### ISSUE TITLE
Version 1.7.2 is not available.

### ISSUE THREAD

**Issue by stewartbryson** [allegro/axion-release-plugin#197]
Even though 1.7.2 shows as the latest release on the Gradle Plugin portal, trying to use that version in a Gradle project fails with:

`Could not find pl.allegro.tech.build:axion-release-plugin:1.7.2.
     Searched in the following locations:
         https://plugins.gradle.org/m2/pl/allegro/tech/build/axion-release-plugin/1.7.2/axion-release-plugin-1.7.2.pom
         https://plugins.gradle.org/m2/pl/allegro/tech/build/axion-release-plugin/1.7.2/axion-release-plugin-1.7.2.jar`

I'm guessing this is actually a Gradle Plugin portal problem, but thought I would raise the issue anyway.

**Comment by adamdubiel** [comment]
I'm using axion 1.7.2 in couple projects already and i see no issues - is it still a problem?

**Comment by stewartbryson** [comment]
Really can't figure this one out. I've tried using 1.7.2 with mulitiple projects, and Gradle just can't find it. As soon as I set it back to 1.7.1... no issues.

FYI... I'm using the new `plugins{}` syntax.

```
==> gradle build
Starting a Gradle Daemon (subsequent builds will be faster)

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'checkmate'.
> Could not resolve all files for configuration ':classpath'.
   > Could not find pl.allegro.tech.build:axion-release-plugin:1.7.2.
     Searched in the following locations:
         https://plugins.gradle.org/m2/pl/allegro/tech/build/axion-release-plugin/1.7.2/axion-release-plugin-1.7.2.pom
         https://plugins.gradle.org/m2/pl/allegro/tech/build/axion-release-plugin/1.7.2/axion-release-plugin-1.7.2.jar
     Required by:
         project :

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

* Get more help at https://help.gradle.org

BUILD FAILED in 9s
==>
```

**Comment by agusmba** [comment]
I got bit by this a few days back.
The problem is that the new version of the plugin is only available in maven central, not in gradle's plugin repo nor in java plugin's default one (jcenter?).
You probably need to add 

```
buildscript {
    repositories {
        mavenCentral()
    }
}
```
to your `build.gradle`

**Comment by agusmba** [comment]
I also find it strange because 1.7.1 was working fine.

If I go with my browser to [plugins.gradle.org](https://plugins.gradle.org/m2/pl/allegro/tech/build/axion-release-plugin/) where gradle is looking for the plugin, I get redirected to [here](https://jcenter.bintray.com/pl/allegro/tech/build/axion-release-plugin/), as you can see, no new versions greater than 1.3.2 there, so that shouldn't have worked for 1.7.1 either.

**Comment by oliverlockwood** [comment]
See https://jcenter.bintray.com/pl/allegro/tech/build/axion-release-plugin/.  It looks like nothing since 1.3.2 has been published to JCenter.

**Comment by adamdubiel** [comment]
First of all, sorry for not taking care of it yet. The reason is - everything works for all my projects, both at work and on Travis CI and i'm currently swamped with work. I should get some breathing space during the weekend and i might look into this.

I just managed to check that nothing changed in publishing configuration for a long long time and thus artifacts disappearing from jcenter would be surprising. But then i'm using official Gradle plugin publisher and i didn't look under the hood for a long time, so sth might have changed.

**Comment by oliverlockwood** [comment]
@adamdubiel it might just be as simple as logging on to bintray and pressing "publish". In my experience publishing artifacts to bintray is a two step process: upload artifacts, then make them publicly available.

**Comment by adamdubiel** [comment]
Unfortunately there are no versions pending publishing on Bintray. I wonder why 1.7.2 is different to other versions.. btw i managed to reproduce this error in (irony is a b***) axion-release :) If i bump axion-release in axion-release to 1.7.2 i see the same results as you do.

**Comment by adamdubiel** [comment]
I thought releasing new version might fix the problem, but apparently it didn't.

However it might be some problem with Gradle plugin portal config itself:

```
curl -iv https://plugins.gradle.org/m2/pl/allegro/tech/build/axion-release-plugin/1.7.0/axion-release-plugin-1.7.0.pom
```

redirects to `plugins-artifacts.gradle.org` and pom is downloaded. However:

```
curl -iv https://plugins.gradle.org/m2/pl/allegro/tech/build/axion-release-plugin/1.7.2/axion-release-plugin-1.7.2.pom
```

also redirects to `plugins-artifacts.gradle.org` but pom is missing. If i enter version that does not exist like `1.9.0`:

```
curl -iv https://plugins.gradle.org/m2/pl/allegro/tech/build/axion-release-plugin/1.8.1/axion-release-plugin-1.9.0.pom
```

I get redirected to `jcenter`, which returns 404.

So looks like Gradle plugin portal is aware of the artifacts i uploaded, but something is wrong with backing store. I will try to submit this issue to Gradle plugin portal and look for help there.

**Comment by adamdubiel** [comment]
**TL;DR**: Use **1.8.1**, works fine.

I did some more diggin (should have done it before releasing 1.8.0 -_- ) and it was outdated plugin-publish-plugin fault.

I was using 0.9.1 version (and could have sworn i updated recently). I updated to 0.9.8 and released 1.8.1 version. Works like a charm now!
